qa/tasks/cephfs/test_volume_client.py

```diff
@@ -34,6 +34,7 @@ def _volume_client_python(self, client, script, vol_prefix=None, ns_prefix=None)
         return client.run_python("""
 from __future__ import print_function
 from ceph_volume_client import CephFSVolumeClient, VolumePath
+from rados import OSError as rados_OSError
```

Comment:
does having a multiple similar exceptions `rados.OSError`, `cephfs.OSError` etc. actually provide any real benefit or is this just a weirdness?